back/src/domains/agency/use-cases/ListAgenciesByFilter.ts

```diff
@@ -85,4 +85,4 @@ export const toAgencyOption = (
 });
 
 const toAgencyOptionName = (agency: AgencyWithUsersRights): string =>
-  `${agency.name} (${agency.address.city})`;
+  `${agency.name}`;
```

Comment:
plus besoin de template littéral
```suggestion
  agency.name
```